src/components/Frame/components/Toast/Toast.scss

```diff
@@ -10,11 +10,11 @@ $Backdrop-opacity: 0.88;
   padding: spacing(tight) spacing();
   border-radius: var(--p-border-radius-wide, border-radius());
   background: var(
-    --p-surface-inverse-foreground,
+    --p-surface-foreground-inverse,
```

Comment:
I'm okay with the name changes here. It actually feels clearer to me what they're for with this change.